Packs/CarbonBlackEnterpriseEDR/TestPlaybooks/Carbon_Black_Enterprise_EDR_Test.yml

```diff
@@ -0,0 +1,699 @@
+id: Carbon Black Enterprise EDR Test
```

Comment:
2 things that miss for me in the test:
 - verify some outputs, the important ones that always return
 - run commands with more than one input to filter